tests/test_geom.py

```diff
@@ -63,7 +64,8 @@ def test_props():
 
     box1copy = geom.box(10, 10, 30, 30, crs=crs)
     assert box1 == box1copy
-    assert box1.convex_hull == box1copy  # NOTE: this might fail because of point order
+    # NOTE: this might fail because of point order
+    assert box1.convex_hull == box1copy
```

Comment:
I often use something like: `(g1^g2).area < tol` in situations like this. That's "XOR" on geometries `g1^g2 <==> (g1-g2)|(g2-g1)`

Reply:
Hmm, I'm not sure where that change came from - I didn't make it intentionally (although I did Black format the entire script...)